presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -1402,7 +1401,7 @@ private void printPlanNodesStatsAndCost(int indent, PlanNode... nodes)
 
         private boolean isKnownPlanNodeStatsOrCost(PlanNode node)
         {
-            return !UNKNOWN_STATS.equals(statsProvider.getStats(node))
+            return !statsProvider.getStats(node).isUnknown()
```

Comment:
Can't we still use `unkown().equals( actual stats )`?

To me, this looks like an unnecessary change in `Add PlanNodeStatsEstimate#unknown() method` commit

Reply:
`unkown().equals( actual stats  aka this)`

This is how `isUnknown` should be implemented

Reply:
Changed the name of the method to `isOutputRowCountUnknown`.